ci/lib/gramerf-groovy-lib.groovy

```diff
@@ -47,6 +47,9 @@ def getNodeName(){
     } else if (run_specific_perf_test.contains("memcached") || run.contains("memcached")){
         echo "memcached workload is selected ..."
         return 'graphene_perf_redis_taken_out_for_vasanth'
+    } else if (run_specific_perf_test.contains("mysql") || run.contains("mysql")){
+        echo "mysql workload is selected ..."
+        return 'graphene_sklearn'
```

Comment:
Is this a typo? Its returning sklearn for mysql?

Reply:
Ohh the node label is sklearn is it?

Reply:
Yes, as we are running this on sklearn machine, returning the corresponding node label.